src/compile.ts

```diff
@@ -385,6 +417,8 @@ export function compile(
           ]);
         } else if (ts.isBreakStatement(node)) {
           return newExpr("BreakStmt", []);
+        } else if (ts.isParenthesizedExpression(node)) {
```

Comment:
Hmm, this is interesting. To ensure precedence is maintained, we should maintain the parentheses with our own `ParanthesizedExpr`.

Reply:
So far the tests seem to show the parenthesis are correctly applied. the contents are already their own expression, curious when the OoO would be incorrect...